Bio/PDB/binary_cif.py

```diff
@@ -0,0 +1,302 @@
+"""
+A module to interact with BinaryCIF-formatted files.
+"""
+
+import gzip
+from collections import deque
+
+import numpy as np
+import Bio.PDB._bcif_helper as _bcif_helper
+
+from Bio.PDB.Structure import Structure
+from Bio.PDB.StructureBuilder import StructureBuilder
+
+try:
+    import msgpack
+except ImportError:
+    from Bio import MissingPythonDependencyError
+
+    raise MissingPythonDependencyError(
+        "Install msgpack to use Bio.PDB.binaryCIF (e.g. pip install msgpack)"
+    ) from None
```

Comment:
Maybe move this above all the PDB imports - i.e. between the third party and local imports?